First_section.html

```diff
@@ -9,53 +9,276 @@
+              <li class="css"><a class="ht" href="#">css</a></li>
+              <li class="javascript"><a class="ht" href="#">javascript</a></li>
+            </ul>
+            <div class="link_to_project">
+              <li class="check_Project"><a href="#">See Project</a></li>
+            </div>
+          </div>
+        </div>
+
+        <div class="work works2">
+          <img src="Snapshoot Portfolio (2).png" alt="snashoot" />
+          <div class="Project info">
+            <h1 class="project_name">Tonic</h1>
+            <ul class="client">
+              <li id="h3">CANOPY</li>
+              <li class="Counter">
+                <img src="Counter.png" alt="counter" />
+              </li>
+
+              <li class="langue">Back end Dev</li>
+              <li class="Counter">
+                <img src="Counter.png" alt="counter" />
+              </li>
+
+              <li class="year">2015</li>
+            </ul>
+            <p class="project_description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="code_language">
+              <li class="html"><a class="ht" href="#">html</a></li>
+              <li class="css"><a class="ht" href="#">css</a></li>
+              <li class="javascript"><a class="ht" href="#">javascript</a></li>
+            </ul>
+            <div class="link_to_project">
+              <li class="check_Project"><a href="#">See Project</a></li>
+            </div>
+          </div>
+        </div>
+
+        <div class="work works3">
+          <img src="Snapshoot Portfolio (3).png" alt="snashoot" />
+          <div class="Project info">
+            <h1 class="project_name">Multi-Post Stories</h1>
+            <ul class="client">
+              <li id="h3">CANOPY</li>
+              <li class="Counter">
+                <img src="Counter.png" alt="counter" />
+              </li>
+
+              <li class="langue">Back end Dev</li>
+              <li class="Counter">
+                <img src="Counter.png" alt="counter" />
+              </li>
+
+              <li class="year">2015</li>
+            </ul>
+            <p class="project_description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="code_language">
+              <li class="html"><a class="ht" href="#">html</a></li>
+              <li class="css"><a class="ht" href="#">css</a></li>
+              <li class="javascript"><a class="ht" href="#">javascript</a></li>
+            </ul>
+            <div class="link_to_project">
+              <li class="check_Project"><a href="#">See Project</a></li>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <div class="About_myself">
+        <section class="top">
+          <h1 id="about">About Myself</h1>
```

Comment:
[Optional]

- [ ] As I mentioned before, please consider using another `hx` tag here. You can refer to this [article](https://www.semrush.com/blog/h1-tag/#h1-tag-best-practices) for more information

Reply:
Done, thank you.